src/server/lib/middleware/loggedInGuard.ts

```diff
@@ -0,0 +1,80 @@
+import { NextFunction, Request } from 'express';
```

Comment:
Could you do a `git-mv` on this file so we can see the changes more clearly. `git-mv` will rename the file without it looking like we've deleted the old one and just created a new one

Reply:
I didn't know about git mv, thank you!